src/main/java/com/buglife/sdk/Buglife.java

```diff
@@ -104,15 +105,29 @@ public static void setUserEmail(@Nullable String userEmail) {
 
     /**
      * Captures a screenshot of the current activity.
+     * Deprecated, use {@link Buglife#captureScreenshot()} instead.
      *
      * @return A bitmap of the generated screenshot
      */
+    @Deprecated
     public static Bitmap getScreenshotBitmap() {
         return getClient().getScreenshot();
     }
 
+    /**
+     * Captures a screenshot of the current activity.
+     *
+     * @return A {@link java.io.File} that represents the screenshot. If screenshot failed, this
+     * will return null.
+     */
+    @Nullable
+    public static FileAttachment captureScreenshot() {
```

Comment:
Should this be `getScreenshotFileAttachment()`, to be more consistent with `addFileAttachment()`? Or is `captureScreenshot` more in line with Java/Android conventions?

Reply:
@schukin I think I prefer `captureScreenshot` because it's performing an action. `getScreenshotFileAttachment` (or any getter) has an implicit meaning of simply retrieving a value/object which is not what this is doing. 

Also, I think the `FileAttachment` class might be better served if it were re-named to `TypedFile` (a file with a mime type). I think the `FileAttachment` name isn't accurate to what it actually is and also bears no useful information. For example,  I think this would read much better overall:

```
TypedFile screenshot = Buglife.captureScreenshot();
Buglife.addAttachment(screenshot);
Buglife.showReporter();
```

What do you think?

Reply:
Cool, I agree w/ `captureScreenshot`. 👍 

IMHO `TypedFile` feels like a better name if it were a private API, but might be confusing for consumers of the public Buglife API. Would prefer to keep it `*Attachment`, since these objects are commonly referred to as "Attachments" in the documentation, the web dashboard & the iOS SDK. (I know, naming for the sake of platform consistency sucks, but this will become more necessary once we add React Native support to Android.)

Also, we'll likely at some point start adding additional data/metadata to attachment objects. In fact, we already do w/ log attachments. :)

Reply:
@schukin I'm ok with leaving the class name as is then.